### PR TITLE
fixed SECURITY-479 https://github.com/jenkinsci/splunk-devops-plugin/3

### PR DESCRIPTION
**Pull request by fengxx**
fixed by https://github.com/jenkinsci/splunk-devops-plugin/pull/3
changes:
https://github.com/jenkinsci/splunk-devops-plugin/compare/jenkinsci:release/1.5.2...1.5.3

### CODE REVIEW

**Comment by daniel-beck** [comment]
Will take a look ASAP. Once merged, don't forget to update https://wiki.jenkins-ci.org/display/JENKINS/Script+Security+Support+in+Plugins

**Comment by fengxx** [comment]
Thanks for the prompt reply!

File: src/main/resources/warnings.json [hunk 1]
```diff
@@ -673,7 +673,8 @@
     "url": "https://jenkins.io/security/advisory/2017-04-10/",
     "versions": [
       {
-        "pattern": ".*"
+        "lastVersion": "1.5.2",
+        "pattern": "1[.][012345](|[.-].*)"
```
Comment (daniel-beck) [inline src/main/resources/warnings.json]: This makes no sense if the fix is in 1.5.3. What am I missing?

Comment (daniel-beck) [inline src/main/resources/warnings.json]: To clarify, the regex is what matters to Jenkins -- `lastVersion` is just for documentation on the plugin site or wiki pages.

Comment (fengxx) [inline src/main/resources/warnings.json]: I thought the check is <= lastVersion using the regex, it seems I am wrong.
how about I bounce the version and release 1.6?

Comment (daniel-beck) [inline src/main/resources/warnings.json]: Jenkins itself matches what's installed to the regex. As this also matches 1.5.3, people with that release will still appear affected. This is a common case, you'll find several patterns dealing with this in the file.

A new version will not make this any more correct, people may be running 1.5.3 and we should not tell them they're unsafe.

**Comment by daniel-beck** [comment]
@fengxx It seems GitHub does not notify about new commits…

**Pull request merged**